### ISSUE TITLE
unlocked cells are still locked in Excel online

### ISSUE THREAD

**Issue by GeorgeHung** [OfficeDev/office-js#143]
The function below protects the active worksheet and keeps cell A1 unlocked.  After running the function, in Windows and Mac, cell A1 are editable.  However, cell A1 is not editable after running the function in Excel online.

``` js
function run() {
  return Excel.run(function(context) {
    var worksheet = context.workbook.worksheets.getActiveWorksheet();
    var range = worksheet.getRange("A1");
    range.format.protection.locked = false;
    worksheet.protection.protect();
    return context.sync().then(function() {});
  });
}
```

## Expected Behavior

Cell A1 should be editable after running the function in Excel online.

## Current Behavior

Cell A1 is not editable after running the function in Excel online.

## Steps to Reproduce, or Live Example

Run the function below.

## Context
In our product we need to have some cells editable while are cells are readonly.

## Your Environment

* Platform [PC desktop, Mac, iOS, Office Online]: PC desktop, Mac, Office Online
* Host [Excel, Word, PowerPoint, etc.]: Excel
* Office version number: 1805 for Windows
* Operating System: Window 8
* Browser (if using Office Online): Chrome

**Comment by JuaneloJuanelo** [comment]
good catch! this is a bug and we are tracking with code 2462836

**Comment by weidezhong-zz** [comment]
As a workaround, add a call to context.sync() immediately after setting locked to false. For example,

range.format.protection.locked = false;
context.sync().then(() => {
      worksheet.protection.protect();
    return context.sync().then(function() {});
}

**Comment by weidezhong-zz** [comment]
The issue should be fixed in build 16.0.10917.37950 now. No workaround is needed anymore.

**Comment by JuaneloJuanelo** [comment]
fixed!!!!
